libsolidity/analysis/ControlFlowBuilder.cpp

```diff
@@ -277,6 +317,55 @@ bool ControlFlowBuilder::visit(FunctionCall const& _functionCall)
 				m_currentNode = nextNode;
 				return false;
 			}
+			case FunctionType::Kind::Internal:
+			case FunctionType::Kind::DelegateCall:
+			{
+				solAssert(!!m_revertNode, "");
+				visitNode(_functionCall);
+				_functionCall.expression().accept(*this);
+				ASTNode::listAccept(_functionCall.arguments(), *this);
+
+				auto const* functionType = dynamic_cast<FunctionType const*>(
+					_functionCall.expression().annotation().type);
+
+				if (!functionType->hasDeclaration())
+					return false;
+
+				auto const* unresolvedFunctionDefinition =
+					dynamic_cast<FunctionDefinition const*>(&functionType->declaration());
+
+				auto const& functionDefinition = m_contract ?
+					unresolvedFunctionDefinition->resolveVirtual(*m_contract) :
+					*unresolvedFunctionDefinition;
+
+				if (!functionDefinition.isImplemented())
+					return false;
+
+				if (m_visitedFunctions.count(&functionDefinition))
+					return false;
+
+				auto& functionFlow = *m_visitedFunctions.emplace(&functionDefinition, initFunctionFlow(m_nodeContainer)).first->second;
+
+				ControlFlowBuilder builder(m_nodeContainer, functionFlow, m_contract, m_visitedFunctions);
+				builder.appendControlFlow(functionDefinition);
+
+				// Function does not have any exit
+				if (!hasExit(functionFlow))
+				{
+					connect(m_currentNode, m_revertNode);
+					m_currentNode = newLabel();
+				}
+				else
+				{
+					auto nextNode = newLabel();
+					connect(m_currentNode, nextNode);
+					m_currentNode = nextNode;
+				}
+
+				return false;
+				break;
+			}
+
```

Comment:
Extra blank

Reply:
separating the default case from the rest seemed like a sensible style..

Reply:
But we usually don't do that. I also don't see the point if the default case does nothing.